pkg/operator/ceph/object/spec.go

```diff
@@ -406,14 +406,24 @@ func (c *clusterConfig) vaultPrefixRGW() string {
 	switch secretEngine {
 	case kms.VaultKVSecretEngineKey:
 		vaultPrefixPath = path.Join(vaultPrefixPath,
-			c.clusterSpec.Security.KeyManagementService.ConnectionDetails[vault.VaultBackendPathKey])
+			c.clusterSpec.Security.KeyManagementService.ConnectionDetails[vault.VaultBackendPathKey], "/data")
 	case kms.VaultTransitSecretEngineKey:
 		vaultPrefixPath = path.Join(vaultPrefixPath, secretEngine, "/export/encryption-key")
 	}
 
 	return vaultPrefixPath
 }
 
+func (c *clusterConfig) sseIsEnabledRGW() bool {
```

Comment:
Nits. 
- It's unclear to me what SSE means. 
- I don't think "RGW" is really needed in the method name. 

Suggest renaming to something like `secretEngineEnabled`.